src/sbt-test/sbt-avro/basic_1.10/test

```diff
@@ -1,4 +1,4 @@
-> set avroValidateDefaults := false
+> set avroSchemaParserBuilder := avroSchemaParserBuilder.value.withValidateDefaults(false)
```

Comment:
This removes tests for deprecated `avroValidateDefaults`, are we sure that they still work, even when deprecated? :)